nemo/collections/tts/models/audio_codec.py

```diff
@@ -32,6 +32,7 @@
     SISDRLoss,
```

Comment:
Will the changes in TTS collection be merged in a separate PR? In that case we should drop them here (CC @zhehuaichen @Edresson @blisc)

Reply:
I talked with @blisc, and we think merging these changes here is fine. I also updated the codec classes to match those in the old codebase.